src/main/java/seedu/address/model/person/ExpiryDate.java

```diff
@@ -0,0 +1,70 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a Food Item's expiry date in Simply Kitchen.
+ * Guarantees: immutable; is valid as declared in {@link #isValidExpiryDate(String)}
+ */
+public class ExpiryDate {
+
+    public static final String MESSAGE_CONSTRAINTS = "Dates should be of the format dd-MM-yyyy or dd/MM/yyyy. ";
```

Comment:
Perhaps "Expiry dates should be of the format d-MM-yyyy, dd-MM-yyyy, d/MM/yyyy or dd/MM/yyyy." might be more accurate?

Reply:
Corrected.